src/parts/Topbar/index.tsx

```diff
@@ -137,7 +138,13 @@ const Topbar = (): JSX.Element => {
             <Tokens />
           </>
         )}
-        <NotificationsPopover address={selectedAccount?.address} items={list} />
+        <div
```

Comment:
Wrapper used here to avoid adding additional CTA/Button variant, or making changes to unrelated components. We'll review this when we update the navigation.